app/src/main/res/layout/opoc_filesystem_item.xml

```diff
@@ -46,6 +48,10 @@
             android:textSize="19sp"
             tools:text="how to write a readme.md" />
 
+        <Space
+            android:layout_width="8dp"
+            android:layout_height="wrap_content" />
+
```

Comment:
Makes default vertical layout 8dp bigger by adding this spacer. If it's needed for horiziontal, hide it in vertical please